providers/os/resources/npm.go

```diff
@@ -322,8 +320,14 @@ func newNpmPackage(runtime *plugin.Runtime, pkg *languages.Package) (*mqlNpmPack
 		mqlFiles = append(mqlFiles, lf)
 	}
 
+	path := ""
+	if len(mqlFiles) > 0 {
```

Comment:
can we make `mqlFiles` to be a `[]*mqlPkgFileInfo{}` list? that way we avoid the casting. not sure if this causes a problem later in the code

Reply:
No,
L335:
```
"files":   llx.ArrayData(mqlFiles, types.Resource("pkgFileInfo")),
```
This requires `[]interfac{}`.